### ISSUE TITLE
GIF images being submitted as PNG

### ISSUE THREAD

**Issue by SparoHawk** [RubaXa/jquery.fileapi#128]
I'm having an issue with the plugin.

When ever I choose a GIF image and send it to the server it is being sent as a PNG.

Has anyone had this issue before?

Thanks.

**Comment by RubaXa** [comment]
Alas, this is not a problem, any transformation of images lead to `png`.

**Comment by SparoHawk** [comment]
Thanks for the answer.

What I want to know is how can I have it send the original GIF image to the server, not a PNG transformation. I tried the "originalImage" option.

**Comment by MisteryX** [comment]
So, how to send on the server original image file without transformation?

**Comment by SparoHawk** [comment]
If you go to RubaXa's other project: [FileAPI](https://github.com/mailru/FileAPI) you will find he uses a totally different `jquery.fileapi`. In this one the images are sent correctly to the server. I thought it was strange since he doesn't do any transformations directly (or at least that I was able to find).

I double checked and found that the FileAPI

**Comment by SparoHawk** [comment]
If you go to RubaXa's other project: [FileAPI](https://github.com/mailru/FileAPI) you will find he uses a totally different `jquery.fileapi`. In this one the images are sent correctly to the server. I thought it was strange since he doesn't do any transformations directly (or at least that I was able to find).

I double checked and found that the `FileAPI` versions here (`jquery.fileapi`) and in `FileAPI` are the same. With odds I tried the other wrapper found in `FileAPI` and it worked. GIFs were uploaded as GIFs.

However, the `jquery.fileapi` wrapper in `FileAPI` has things missing that this one has. Like the `maxFiles` option and the `beforeUpload` event.
